src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/reorg_yolo_gpu_ref.cl

```diff
@@ -41,25 +59,42 @@ KERNEL (reorg_yolo_ref)(const __global UNIT_TYPE* input, __global UNIT_TYPE* out
 
         output[dstIndex] = input[srcIndex];
     }
-#elif OUTPUT_LAYOUT_YXFB
-    int ic = get_global_id(0) / B;
-    int ib = get_global_id(0) % B;
-    int ih = get_global_id(2);
-    int iw = get_global_id(1);
+#else
+    const uint ic = get_global_id(2);
+    const uint ih = get_global_id(1);
+    const uint iw = get_global_id(0);
```

Comment:
```
    const uint ic = get_global_id(2);
    const uint ih = get_global_id(1);
    const uint iw = get_global_id(0);
```

This part seems to be similar for both branches

Reply:
Done.